plugins/chat/app/controllers/chat/incoming_webhooks_controller.rb

```diff
@@ -0,0 +1,113 @@
+# frozen_string_literal: true
+
+module Chat
+  class IncomingWebhooksController < ::ApplicationController
```

Comment:
Not really related to this PR, but it doesn't look like this controller checks for `SiteSetting.chat_enabled` ?

Reply:
Will check this in follow up PR, good call thanks